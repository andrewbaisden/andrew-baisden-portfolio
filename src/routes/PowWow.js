import React from 'react';
import { Helmet } from 'react-helmet';
import Header from '../components/Header';
import Footer from '../components/Footer';
import PortfolioPagePowWow from '../components/PortfolioPagePowWow';
import CheckPortfolioPage from '../components/CheckPortfolioPage';
// import PortfolioCarouselPowWow from '../components/PortfolioCarouselPowWow';

const Sky = () => (
  <div>
    <div>
      <Helmet>
        <meta
          name="description"
          itemProp="description"
          content="An agency approached me with a job working at PowWow who are a multimedia creative agency. It was going to be a two week contract."
        />
        <title>PowWow - Portfolio - Full Stack Developer | Andrew Baisden | London</title>
        <link rel="canonical" href="https://andrewbaisden.com/portfolio/powwow" />
      </Helmet>
    </div>
    <CheckPortfolioPage />
    <Header />
    <PortfolioPagePowWow companyBG="#ffffff" />
    {/* <PortfolioCarouselPowWow /> */}
    <Footer />
  </div>
);

export default Sky;
