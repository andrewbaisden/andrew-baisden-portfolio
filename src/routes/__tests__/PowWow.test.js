import React from 'react';
import { shallow } from 'enzyme';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import PowWow from '../PowWow';

Enzyme.configure({ adapter: new Adapter() });

describe('<PowWow />', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<PowWow />);
    expect(wrapper).toMatchSnapshot();
  });
});
