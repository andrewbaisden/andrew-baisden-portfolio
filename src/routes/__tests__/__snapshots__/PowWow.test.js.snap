// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PowWow /> renders and matches the snapshot 1`] = `ShallowWrapper {}`;
